src/main/java/org/betonquest/betonquest/compatibility/protocollib/wrappers/WrapperPlayServerScoreboardTeam.java

```diff
@@ -0,0 +1,248 @@
+package org.betonquest.betonquest.compatibility.protocollib.wrappers;
+
+import com.comphenix.protocol.PacketType;
+import com.comphenix.protocol.utility.MinecraftReflection;
+import com.comphenix.protocol.wrappers.WrappedChatComponent;
+import org.bukkit.ChatColor;
+
+import java.util.Collection;
+import java.util.List;
+
+@SuppressWarnings({"PMD.CommentRequired", "PMD.UnusedPrivateField"})
+public class WrapperPlayServerScoreboardTeam extends PacketHandlerDecorator {
```

Comment:
This class is already implemented in the dependency PacketWrapper. There should be no need to copy paste this class into BetonQuest